### ISSUE TITLE
new schema_force_visible column

### ISSUE THREAD

**Issue by tudorbarascu** [qwat/qWat#18]
Good morning,
In the qgs project, the pipe styles still take into consideration the  old "_schema_visible" column which doesn't exist anymore. I can make a pull request if you hadn't already changed the project locally.

**Comment by 3nids** [comment]
ok, I'm fixing it.

**Comment by 3nids** [comment]
there are still some remaining _schema_visible but they are in the "styles" part of the project file. I can't access them through QGIS application. I don't know how to update them.

**Comment by tudorbarascu** [comment]
I did a find and replace of "_schema_visible =" with "schema_force_visible =" on the qgs project. What do you think?

**Comment by 3nids** [comment]
That's not correct.
_schema_visible has been simply removed. It is now calculated using `COALESCE( schema_force_visible, xxx_function.schema_visible )`

`schema_force_view` has been renamed to `schema_force_visible` for consistency.

The overall idea is that the fact that an item (pipe, valve) is visible or not at small scales depends on its function. However, it can be overwritten by  the `schema_force_visible` field (for more flexibility).

So, basically `_schema_visible` field does not exist anymore and should not be present in qgis project. I don't understand why it remains in some parts of the project but I suppose it's a bug with the new layer styles code.

**Comment by tudorbarascu** [comment]
You're right.
For my project which differs a little the _schema_visible part still remains as I have joined tables with columns that are named "schema_visible" and the join puts by default ``_`` before the column name.
